### PR TITLE
Review Pietro

### LINKED ISSUE #16: Translating technical footnote
I was not able to translate this footnote as I don't really know much about the domain.

> [^outgoing]: “The overall 3.4 fold reduction [70% reduction] in aerosol copy numbers we observed combined with a nearly complete elimination of large droplet spray demonstrated by Johnson et al. suggests that surgical masks worn by infected persons could have a clinically significant impact on transmission.” [Milton DK, Fabian MP, Cowling BJ, Grantham ML, McDevitt JJ](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3591312/)

### PR DESCRIPTION
**Pull request by pietroppeter**
PR ready for review.

In the end I reverted the big change on the title and we these are the changes:

* added mention of Franklin D Roosevelt in initial quote. I think for the Italian reader the fact that the quote is from FDR is not self evident. We could add a footnote explaining the context but I would avoid that (it would be a NdT footnote). The curious reader can google the context of FDR quote to discover that it was done after the Great Depression, while launching the New Deal. See for example [here ](https://library.weschool.com/lezione/franklin-delano-roosevelt-new-deal-john-maynard-keynes-storia-riassunto-17774.html )and [here](https://library.weschool.com/lezione/franklin-delano-roosevelt-new-deal-john-maynard-keynes-storia-riassunto-17774.html).
* added a commit on metadata for Open Graph, following suggestion by @grgfgh. This is the only changed I did both in index_head and index.html (the rest of changes will have to go through the build process to be visible)
* added a commit on numero di riproduzione "netto" instead of effetivo (still mentioning the alternative). I would keep "netto" since it is officially mention in the "bollettino" on Rt that ISS (Istituto Superiore di Sanità) started to publish and which will be updated weekly. This is by the way a very interesting publication that I was not able to find mentioned in many places (it reports for example estimates of Rt by region). The second issue is [this one](https://www.epicentro.iss.it/coronavirus/bollettino/Bollettino-sorveglianza-integrata-COVID-19_7-maggio-2020.pdf) (check page 15 for the note on Rt) and you can find the subsequent ones by following [this page](https://www.epicentro.iss.it/coronavirus/sars-cov-2-sorveglianza-dati).
* added a commit for the translation of technical footnote (will close #16). Chose not to translate "droplet" since it is now commonly used, see e.g. [here](https://www.lastampa.it/cronaca/2020/03/02/news/cosa-vuol-dire-droplet-e-perche-c-entra-con-la-distanza-che-dobbiamo-tenere-dalle-persone-infette-1.38541450) and [here](https://www.ilpost.it/2020/05/12/contagi-coronavirus-luoghi/).
* added commit for #11

### CODE REVIEW

**Comment by simonefalcini** [comment]
Oh I missed this, I translated the thumbnail to "cosa succederà dopo? - i possibili futuri del covid-19 spiegati con simulazioni interattive"
If you prefer "e ora che succede" I will change the translation and update the file.

**Comment by pietroppeter** [comment]
Hi @simonefalcini sorry that's my bad you missed it, @harisont had warned me to add a reference in the main translation discussion and I planned to but had not gotten there yet. Anyway I do not think it is yet settled the switch to "E ora che succede?", I would like to have feedback from you @harisont, @mendaomn and anyone who might want to pitch in. If overall we find positive or neutral feelings about it, yes, I would prefer to change it to "E ora che succede?".

**Comment by pietroppeter** [comment]
Ah, maybe you missed #13 too (😉), but I think tracking image translations could be useful in this proofreading phase, if you have bandwidth at the moment.

File: index_head.html [hunk 1]
```diff
@@ -4,20 +4,20 @@
...
+	<meta name="twitter:title" content="E ora che succede? Futuri possibili per COVID-19, spiegati con simulazioni interattive">
 	<meta name="twitter:description" content="una guida interattiva">
 	<meta name="twitter:image" content="https://ncase.me/covid-19/sharing/thumbnail.png">
 
 	<!-- Open Graph data -->
-	<meta property="og:title" content="E adesso cosa succede? Futuri possibili per COVID-19, spiegati con simulazioni interattive">
+	<meta property="og:title" content="E ora che succede? Futuri possibili per COVID-19, spiegati con simulazioni interattive">
 	<meta property="og:type" content="website">
 	<meta property="og:url" content="https://ncase.me/covid-19/">
 	<meta property="og:image" content="https://ncase.me/covid-19/sharing/thumbnail.png">
```
Comment (grgfgh) [inline index_head.html]: I believe this should be: `<meta property="og:image" content="https://harisont.github.io/covid-19/sharing/thumbnail.png">`

Comment (grgfgh) [reply index_head.html]: also the og:url is referencing the English version

Comment (mendaomn) [reply index_head.html]: Absolutely true, would you like to submit those changes?

**Comment by mendaomn** [comment]
> Hi @simonefalcini sorry that's my bad you missed it, @harisont had warned me to add a reference in the main translation discussion and I planned to but had not gotten there yet. Anyway I do not think it is yet settled the switch to "E ora che succede?", I would like to have feedback from you @harisont, @mendaomn and anyone who might want to pitch in. If overall we find positive or neutral feelings about it, yes, I would prefer to change it to "E ora che succede?".

I'm pretty neutral about it, let's do it if it makes more sense to you!

**Comment by simonefalcini** [comment]
I agree: let's stick to "E ora che succede?" and use it everywhere so that we can move forward :)
If you agree, I'll change the thumbnail

**Comment by pietroppeter** [comment]
In the end I decided that “E adesso cosa succede” was fine. I will revert the commit, can you change the thumbnail that way?

**Comment by simonefalcini** [comment]
> In the end I decided that “E adesso cosa succede” was fine. I will revert the commit, can you change the thumbnail that way?

sure thing, doing it now on master

**Review by mendaomn — COMMENTED** [review]
Looks good to me!

File: words/words.md [hunk 2]
```diff
@@ -28,7 +28,7 @@
 	"The only thing to fear is fear itself" was stupid advice.
 -->
 
-"L'unica cosa di cui avere paura è la paura stessa" è un consiglio stupido.
+"L'unica cosa di cui avere paura è la paura stessa", disse Franklin D. Roosevelt. Era un consiglio stupido.
```
Comment (harisont) [inline words/words.md:31]: Mmm should we add that?

**Review by harisont — COMMENTED** [review]
Only thing I feel perplexed about is the quote in the beginning, why the addition?

**Comment by pietroppeter** [comment]
I think the average Italian reader would miss where the quote comes from, while the average US reader would not.

In this case I think it is better to add context, otherwise you will lose the meaning of the sentence: “remember when a famous and well respected politician during a moment of crisis gave this advice that started a new renaissance period? Well that was crappy advice and do not get politicians of today give you the same...”.

The first sentence is important and and they quote it back at the end. If you do not get that this is important quote you might end up thinking: “why are they making such a fuss from this detabable quote that they decided to start with?”. Because the context around the quote is important.

I think I also saw other translations do that.I will try to find reference.

**Comment by pietroppeter** [comment]
I checked, Spanish and Portuguese do not have it. German has a footnote (I was tempted to add a footnote explaining better the context, that would have meant also moving first footnote content above, but decided it would have interrupted the flow and that it is better just to give a hint of context for users). French translation (not yet official) has it.

**Comment by pietroppeter** [comment]
solved two small conflicts.

**Comment by harisont** [comment]
Well I think it's reasonable to have it. 
I'll merge this, update the html and turn our draft pull request on Nicky's repo into one ready for merge :)

**Pull request merged**